csm/conf/etc/csm/csm.conf

```diff
@@ -191,3 +191,8 @@ STORAGE_CAPACITY_MANAGMENT:
 RETRY:
   retry_count: 5
   sleep_duration: 3
+TOPLOLOGY:
+    backend: "consul"
+    index: "conf"
+    object_store: "cortx"
```

Comment:
Are you maintaining default_csm.conf  with new additions ?

Reply:
yes added